services/py-api/README.md

```diff
@@ -1,7 +1,7 @@
 ## How to set up the API and run it as a standalone:
 
 ### Prerequisites:
-1. Have [Python3.13](https://www.python.org/downloads/release/python-3130/) installed. You could use [pyenv](https://github.com/pyenv/pyenv?tab=readme-ov-file#a-getting-pyenv)
```

Comment:
The link still points to 3.13, but you know what, just remove this README, as it is no longer needed, due to the dev container.